css/_menus.scss

```diff
@@ -6,7 +6,7 @@
 //
 //Markup:
 //  <h3>Button menu trigger:</h3>
-//  <div tabindex=0 class="context-menu wfm-btn wfm-btn-default">
+//  <div tabindex=0 class="context-menu context-btn wfm-btn wfm-btn-default">
```

Comment:
Is the .context-btn always a wfm-btn and/or wfm-btn-default? Could it extend those classes ? It would avoid having to write all of them.